scrapeIDArtists/server.js

```diff
@@ -0,0 +1,86 @@
+const http = require('http');
+var request = require("request");
+var fs = require('fs');
+var async = require('async');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hello World\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Server running at http://${hostname}:${port}/`);
+
+  var obj = {
+    table: []
+  };
+
+  // obj.table.push({id: 1, type:"person", name:"Lukas Music Star", country: "LT"});

```

Comment:
Why is there commented code in this PR? What's the purpose of this code, why is it not uncommented, is it needed?